RELEASE_PROCESS.md

```diff
@@ -36,13 +36,12 @@ The result of that workflow is the creation and publishing of a Docker image wit
 
 ### Create an official `Release` in GitHub
 
-Once the tag has been pushed to the repository and the build has been verified, a `Release` should be created within GitHub matching the tag.  This is a manual step that must be completed after the automation.
+Once the tag has been pushed to the repository and the build process completed, new `Release` will be created automatically within GitHub matching the tag. Manual step is required to update release notes.
```

Comment:
Minor rewording to add some missing articles: 
`Once the tag has been pushed to the repository and the build process completed, new Release will be created...` --> `Once the tag has been pushed to the repository and the build process completed, a new Release will be created...` 
`Manual step is required to update release notes` --> `A manual step is required to update the release notes`